### ISSUE TITLE
Add support for Windows 2016 and 2012

### ISSUE THREAD

**Issue by tuxmea** [example42/puppet-tp#49]

**Comment by alvagante** [comment]
We manage packages here https://github.com/example42/puppet-tp/blob/master/manifests/install.pp#L241 this should be enough to manage packages on Windows , with or without Chocolatey, if there's the relevant data in TinyData.

Services are managed here:
https://github.com/example42/puppet-tp/blob/master/manifests/install.pp#L252
and also this should be enough to manage services under Windows.

Files are managed in different places, in the main tp::conf define  we should have whataver is needed to manafe configuration files under Windows: 
https://github.com/example42/puppet-tp/blob/master/manifests/conf.pp#L324 

So, basically, Windows support should already be possible with tp, at least for the core tp::install, tp::conf and tp::dir defines. It's proably mostly a matter of having the correct relevant tinydata.

**Comment by alvagante** [comment]
Tp uses standard package service and file resource types. As long as correct tinydata is present (and chocolatey is used) Windows is supported.
Note anyway that the tp command, which can be optionally installed on a system, doesn't work on Windows.
